spec/abstractops.html

```diff
@@ -503,21 +503,29 @@ <h1>ToRelativeTemporalObject ( _options_ )</h1>
         1. Let _result_ be ? InterpretTemporalDateTimeFields(_calendar_, _fields_, _dateOptions_).
         1. Let _offsetString_ be ? Get(_value_, *"offset"*).
         1. Let _timeZone_ be ? Get(_value_, *"timeZone"*).
+        1. If _timeZone_ is not *undefined*, then
+          1. Set _timeZone_ to ? ToTemporalTimeZone(_timeZone_).
         1. If _offsetString_ is *undefined*, then
           1. Set _offsetBehaviour_ to ~wall~.
       1. Else,
         1. Let _string_ be ? ToString(_value_).
         1. Let _result_ be ? ParseTemporalRelativeToString(_string_).
         1. Let _calendar_ be ? ToTemporalCalendarWithISODefault(_result_.[[Calendar]]).
         1. Let _offsetString_ be _result_.[[TimeZoneOffset]].
-        1. Let _timeZone_ be _result_.[[TimeZoneIANAName]].
+        1. Let _timeZoneName_ be _result_.[[TimeZoneIANAName]].
+        1. If _timeZoneName_ is not *undefined*, then
+          1. If ParseText(! StringToCodePoints(_timeZoneName_), |TimeZoneNumericUTCOffset|) is not a List of errors, then
```

Comment:
This use of ParseText is very different than other part of the spec parsing the ISO 8601 grammar. Is there a reason for that? or could it be rewrite to use similar wording like other parsing part?
Is the line the same as 
"If _timeZoneName_ does satisfy the syntax of a TimeZoneNumericUTCOffset (see 13.33), then "
as similar (removed the "not") used in https://tc39.es/proposal-temporal/#sec-temporal-parsetimezoneoffsetstring